content/en/docs/reference/glossary/qos-class.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QoS Class
+id: qosclass
```

Comment:
Just realised, we normally put a dash between the words in a glossary entry's `id`.